crates/python/src/api.rs

```diff
@@ -0,0 +1,223 @@
+use std::collections::HashMap;
```

Comment:
Moved this stuff from `lib.rs` to `api.rs`. It is re-exported at the top level to avoid breakages.